### ISSUE TITLE
week view: require a dynamical way to change the data in footer without refreshing whole widget

### ISSUE THREAD

**Issue by infor-mahesh** [infor-design/enterprise#8237]
**Is your feature request related to a problem or use case? Please describe.**
Week view component: Changing the footer value with some specific conditions whether it needs to be visible or not, so we are configuring it with some parameters but to reflect those changes in the footer it requires a browser refresh, which is not the case for the feature we require

**Describe the solution you'd like**
so we require something that if we change any variable in runtime the footer value needs to be updated without the need of refreshing complete widget

**Describe alternatives you've considered**
This is a blocker which needs to be resolved

**Additional context**
And what application do you work on? [e.g. {Infor XYZ}]

**Comment by tmcconechy** [comment]
@infor-mahesh not sure i follow. If i go to https://main-enterprise.demo.design.infor.com/components/week-view/example-stacked.html and edit the div/span with the text in it it can be updated.

Cant you just change the DOM element or attach it to a template?

**Comment by infor-mahesh** [comment]
yes but after loading the widget and when the div or span what your refering if updated then it is not dynamically changing over widget it requires a refresh to get the changes updated

**Comment by tmcconechy** [comment]
@infor-mahesh not sure we can fix this without an example. Can you please put a runnable example on the issue. You can either

1) Explain how to do it in our examples https://github.com/infor-design/enterprise-ng or https://github.com/infor-design/enterprise
2) Post the code and steps in https://stackblitz.com/edit/ids-quick-start-1721-mn9sys or https://stackblitz.com/edit/ids-enterprise-4991?file=index.js
3) Post some other runnable code (anything where we can run and see the issue)

All issues we get require steps to reproduce and a runnable example or some kind

**Comment by infor-mahesh** [comment]
Hi @tmcconechy , Will try the DOM operation once again, if it didn't work will provide a reproducible example. thankyou
